Jobsway2goMvc/Controllers/EventsController.cs

```diff
@@ -336,5 +350,24 @@ public IActionResult GuestsSelect(int eventId)
                                .Where(i => i.EventId == eventId);
             return View(eventGuests);
         }
-    }
+
+
+        private string UploadedFile(Event model)
+            {
+                string uniqueFileName = null;
+
+                if (model.ImagePath != null)
+                {
+                    string uploadsFolder = Path.Combine(_hostEnvironment.WebRootPath, "images");
+                uniqueFileName = Guid.NewGuid().ToString() + "_" + model.ImageName.FileName;
+                    string filePath = Path.Combine(uploadsFolder, uniqueFileName);
+                    using (var fileStream = new FileStream(filePath, FileMode.Create))
```

Comment:
`uniqueFileName` will throw Null Reference as the `ImageName` property is null and the name of the file is stored in ImagePath instead.